### DISCUSSION TITLE
Is their a plan using Vite for the vue-cli, for faster development and faster build time ?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuejs/vue-cli#6361] [Q&A]
So one of the things that developers like to have is the power to build or serve a project in a very short amount of time.. For small projects, this is clearly cannot be observed, but when projects become bigger and bigger the dev serve, and build time is a little bit slow. Will the CLI will adapt to the viteenvironment?

**Comment by laurensiusadi** [comment]
In my own opinion, I don't think vue-cli itself will move to use Vite. You can refactor your project to use Vite.

**Reply by JenuelDev** [reply]
I see, thanks.. yeah, I think vue-CLI is just fine for me.. :) I was able to learn to use vite in my vue cli project... so think, I agree to your point :)

**Comment by Viggo95** [comment]
Maybe you can try using this👇，it might help you
[vue-cli-plugin-vite](https://github.com/IndexXuan/vue-cli-plugin-vite)

**Reply by JenuelDev** [reply]
Oh nice thanks thanks :)
